adoc/i-Introduction/SA.adoc

```diff
@@ -1,40 +1,42 @@
+When deploying and managing a large number of systems in an SAP Landscape, there is often a secondary need, getting an overview of what is happening in he environemt after the install is complete.
 
-An additional challenge is to make the infrastructure high available, as this will add more complexity and tasks.
+To help with this, SUSE has added the ability gain insight into your SAP Landscape with a comprehensive monitoring solution, providing dashboards, realtime and historic views and active alerts and reporting.
 
-Such a number of systems in SAP Landscapes have an further need - getting an overview whats going on after the install is done. Therefore we added the possibility to allow insights into your SAP Landscape with comprehensive dashboards, realtime and historic views and active alerts and reporting.
 
-The major motivation was to improve, simplify and unify the installation of SAP Landscape on SUSE Linux Enterprise Server for SAP Applications and clearly standardize deployments and allow customers to use one level of tooling in various ways – from a Command Line interface, through some GUI driven process and SUSE Manager or other automation frameworks.
+The key goals are:
+- improve, simplify and unify the installation of SAP Landscape on SUSE Linux Enterprise Server for SAP Applications 
+- clearly standardize deployments and allow customers to use one set of tooling for multiple use cases such as Command Line interface, a GUI driven process, SUSE Manager or other automation framework.
 
-So it was clear for us to move to a more modern approach, like infrastructure-as-code, in order to help our customers to reduce the effort and errors.
+To achieve this, SUSE have adopted a more modern approach, infrastructure-as-code which helps customers reduce the effort and errors during deployments.
 
-As SUSE Linux Enterprise Server and many other SUSE products ship with a universal configuration management solution since a few years, we used it as the base for the new automation. 
+In recent years, SUSE Linux Enterprise Server and many other SUSE products shipped with a universal configuration management solution, this is used as the foundation for the new automation capability. 
 
-This configuration infrastructure management system is called Salt from SaltStack and provides a highly scalable, powerful and fast infrastructure automation and management, built on a dynamic communication bus.
-Salt can be used for data-driven orchestration, remote execution for any infrastructure, configuration management for any app stack, and much more.
+This configuration management system is called Salt from SaltStack and provides a highly scalable, powerful and fast infrastructure automation and management, built on a dynamic communication bus.
+Salt can be used for data-driven orchestration, remote execution for any infrastructure, configuration management for any app stack.
```

Comment:
it could be used for more, e.g. security, compliance, ... so why you removed the "much more"?

Reply:
Added back in.